nano/core_test/request_aggregator.cpp

```diff
@@ -111,17 +111,17 @@ TEST (request_aggregator, one_update)
 	node.aggregator.request (request, dummy_channel);
 	// In the ledger but no vote generated yet
 	ASSERT_TIMELY (3s, 0 < node.stats.count (nano::stat::type::requests, nano::stat::detail::requests_generated_votes))
-	ASSERT_TRUE (node.aggregator.empty ());
+	ASSERT_TIMELY (3s, node.aggregator.empty ());
 	ASSERT_TIMELY_EQ (3s, 2, node.stats.count (nano::stat::type::aggregator, nano::stat::detail::aggregator_accepted));
 	ASSERT_TIMELY_EQ (3s, 0, node.stats.count (nano::stat::type::aggregator, nano::stat::detail::aggregator_dropped));
 	ASSERT_TIMELY_EQ (3s, 0, node.stats.count (nano::stat::type::requests, nano::stat::detail::requests_unknown));
 	ASSERT_TIMELY_EQ (3s, 2, node.stats.count (nano::stat::type::requests, nano::stat::detail::requests_generated_hashes));
 	size_t count = 0;
-	ASSERT_TIMELY_EQ (3s, 1, (count = node.stats.count (nano::stat::type::requests, nano::stat::detail::requests_generated_votes)));
+	ASSERT_TIMELY_EQ (3s, 2, (count = node.stats.count (nano::stat::type::requests, nano::stat::detail::requests_generated_votes)));
```

Comment:
This should probably check for greater or equal to one. Not sure how exactly it's implemented in vote generator, but depending on timing, two hashes could be bundled into one vote.

Reply:
In this testcase, it always returns 2 but I can change it to > 1.
Should we have another testcase that explicitly tests this bundling ?

Reply:
I think it's tested somewhere. It might be always 2 just because the load is light, not sure.

Reply:
I've seen you've fixed this test already in a recent commit. I'll close this one

Reply:
created a new one without conflicts that addresses the remaining issue: https://github.com/nanocurrency/nano-node/pull/4622